build_update.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Update (or install) build-tools to latest release on https://github.com/drud/build-tools/releases/latest
+
+set -e
+
+base_url=$(curl -s -I https://github.com/drud/build-tools/releases/latest | awk '/^Location/ {gsub(/[\n\r]/,"",$2);  printf "%s", $2}')
+tag=${base_url##*/}
+
+tarball_url="https://github.com/drud/build-tools/archive/$tag.tar.gz"
+internal_name=build-tools-$tag
+local_file=/tmp/$internal_name.tgz
+
+# If there is a current build-tools, get permission and remove
+if [ -d ../build-tools ]; then
+	echo "OK to replace current build-tools at $PWD?"
+	read -p "Replace build-tools with latest version? y/N" -n 1 -r
+	echo
+	if ! [[ $REPLY =~ ^[Yy]$ ]]
+	then
+	  echo "Exiting"
+	  exit 1
+	fi
+	cd ..
+	git rm -r build-tools
+# If no current build-tools, prompt, get permission to add
+else
+	echo "OK to add current build-tools at $PWD/build-tools?"
+	read -p "Add build-tools with latest version? y/N" -n 1 -r
+	echo
+	if ! [[ $REPLY =~ ^[Yy]$ ]]
+	then
+	  echo "Exiting"
+	  exit 1
+	fi
+fi
+
+
+wget -q -O $local_file $tarball_url
+tar -xf $local_file
+mv $internal_name build-tools
+git add build-tools
+echo "Updated build-tools to $tag
+
+$base_url" | git commit -F -
```

Comment:
I'm not sure if I love the commit here or not. I can definitely see an argument for not doing it.